### ISSUE TITLE
Don't require a leading v when switching versions

### ISSUE THREAD

**Issue by hdgarrood** [ThomasCrevoisier/psvm-js#10]
For example:

```
$ psvm use 0.8.5
Switching to Purescript : 0.8.5
Unhandled rejection Error: ENOENT: no such file or directory, open '/Users/hgarrood/.psvm/versions/0.8.5/purescript/psc'
    at Error (native)
```

I do have 0.8.5 installed; if I run `psvm use v0.8.5` it works. Can `psvm` automatically prepend a 'v' if necessary? Also can this error be made a bit nicer?

I might send a PR for this if I get a spare moment. :)

**Comment by ThomasCrevoisier** [comment]
Yep, you're absolutely right. I realized that there is some work to do on the error to display more friendly messages ^^

**Comment by ThomasCrevoisier** [comment]
@hdgarrood I took a look, and wanted to do like `nvm` (ie : if you type `psvm install 0.10` it will install the latest version matching `0.10`).
Using `node-semver` did the job, and handles also the leading `v`.
The only problem I encouter is that if I type `psvm install 0.10`, the `0.10` will actually be parsed by `minimist` and give `0.1`.
I did not find any good fix for the moment. Maybe you have some ideas ?

**Comment by hdgarrood** [comment]
Hm, not sure. Can you show me the code you have so far?

**Comment by ThomasCrevoisier** [comment]
Here is my attempt on the download part :
https://gist.github.com/ThomasCrevoisier/a3aa3038d60a04c6b1c7746c6d696e09

**Comment by felixSchl** [comment]
Would `opts.string` do the trick? https://github.com/substack/minimist#var-argv--parseargsargs-opts
